content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,250 @@
+   the API defines _Core_, _Extended_, and _Implementation-specific_ support levels for each feature:
+
+   * __Core__ feature is portable across ALL implementations and is validated by conformance tests. A Core
+     feature is part of the API types schema.
+   * __Extended__ feature is portable but not universally supported across implementations. Implementations
+     that support the feature will have the same behavior and semantics. It is expected that some number of
+     roadmap features will eventually migrate into the _Core_. An _Extended_ feature is part of the API types
+     schema.
+   * __Implementation-specific__ feature is not portable across implementations and is vendor-specific.
+     An _Implementation-specific_ feature will not have API types and schema, except via generic extension
+     points. The implementation is responsible for providing conformance tests to validate these features.
+
+1. __Conformance Tests:__ Gateway API includes a set of conformance tests to validate _Core_ and _Extended_
+   features.
+
+## Getting started
+
+Instead of Gateway API resources being natively implemented by Kubernetes, the specifications
+are defined as [Custom Resources][crds] supported by a wide range of [implementations][implementations].
+You can [install][install] the Gateway API CRDs manually, but this step is typically accomplished by
+your selected implementation. Use one of the following tabs to install a Gateway API implementation:
+
+{{< tabs name="service_tabs" >}}
+{{% tab name="Default" %}}
+Select one of the tabs.
+{{% /tab %}}
+
+{{% tab name="Envoy Gateway" %}}
+
+Getting started with Envoy Gateway requires [Helm][helm] and [kubectl][kubectl].
+
+Install the Gateway API CRDs and Envoy Gateway:
+
+```sh
+helm install eg oci://docker.io/envoyproxy/gateway-helm --version v0.5.0 -n envoy-gateway-system --create-namespace
+```
+
+Wait for Envoy Gateway to become available:
+
+```sh
+kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
+```
+
+Install the GatewayClass, Gateway, HTTPRoute, and example application resources:
+
+```sh
+kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/v0.5.0/quickstart.yaml -n default
+```
+
+Follow [these steps][eg-testing] to test the installation or refer to the official Envoy Gateway
+[documentation][eg-docs] for additional instructions.
+
+[eg-docs]: https://gateway.envoyproxy.io/
+[eg-testing]: https://gateway.envoyproxy.io/v0.5.0/user/quickstart/#testing-the-configuration
+[helm]: https://helm.sh/docs/intro/install/
+[kubectl]: https://kubernetes.io/docs/tasks/tools/#kubectl
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+{{< note >}}
+Make sure to review the documentation of your selected implementation to understand any caveats.
+{{< /note >}}
+
+Refer to the Gateway API [guides][guides] to learn more about getting started.
+
+## Migrating from Ingress
+
```

Comment:
small suggestion: if adding in `What's Next` section, maybe move this under that